src/OpenTK-1.0/OpenTK.csproj

```diff
@@ -599,6 +599,46 @@
         </Compile>
    </ItemGroup>
   <Import Project="$(OutputPath)\..\..\..\xbuild\Xamarin\Android\Xamarin.Android.CSharp.targets" />
+  <PropertyGroup>
+    <OpenTKAssembly>$(OutputPath)$(AssemblyName).dll</OpenTKAssembly>
+    <OpenTKAssembly Condition="!Exists('$(OpenTKAssembly)')">..\..\bin\$(Configuration)\lib\xbuild-frameworks\MonoAndroid\$(AndroidLatestFrameworkVersion)\$(AssemblyName).dll</OpenTKAssembly>
+  </PropertyGroup>
+  <Target Name="_GenerateOpenTKJavaCallableWrappers"
```

Comment:
This method looks ~identical to the [`_GenerateJavaCallableWrappers` target](https://github.com/xamarin/xamarin-android/blob/c76fc64/src/Mono.Android/Mono.Android.targets#L136-L170).

Would it be possible to "merge" these targets into a `GenerateJavaCallableWrappers` target in e.g. `build-tools/scripts/JavaCallableWrappers.targets`, operating on an `@(JavaCallableWrapperSource)` item group?